.github/workflows/windows.yml

```diff
@@ -117,3 +119,68 @@ jobs:
         run: |
           cd tools
           python -m pytest --pyargs numpy -m "not slow" -n2 --timeout=600 --durations=10
+
+  #=======================================================================================
+  msvc_python64bit_openblas:
+    name: MSVC, ${{ matrix.TEST_MODE }}, BLAS${{ matrix.BLAS }}
+    runs-on: windows-2022
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - architecture: x64
+            BLAS: 64
+            TEST_MODE: full
+            pyver: '3.12'
+          - architecture: x64
+            BLAS: 32
+            TEST_MODE: fast
+            pyver: '3.11'
+
+    # To enable this job on a fork, comment out:
+    if: github.repository == 'numpy/numpy'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          submodules: recursive
+          fetch-tags: true
+          persist-credentials: false
+
+      - name: Setup Python
+        uses: actions/setup-python@e797f83bcb11b83ae66e0230d6156d7c80228e7c # v6.0.0
+        with:
+          python-version: ${{ matrix.pyver }}
+          architecture: ${{ matrix.architecture }}
+
+      - name: Setup MSVC
+        uses: bus1/cabuild/action/msdevshell@e22aba57d6e74891d059d66501b6b5aed8123c4d  # v1
+        with:
+          architecture: ${{ matrix.architecture }}
+
+      - name: pkg-config, rtools
+        run: |
+          choco install -y --stoponfirstfailure --checksum 6004DF17818F5A6DBF19CB335CC92702 pkgconfiglite
+
+      - name: Dependencies
+        run: |
+          python -m pip install --upgrade pip wheel
+          python -m pip install -r requirements/test_requirements.txt
```

Comment:
It may be nice to pass `-Csetup-args=-Dallow-noblas=false` here to prevent regressions. `scipy-openblas` has been known to go silently missing on occasion.